charts/zipkin/Chart.yaml

```diff
@@ -14,7 +14,7 @@ appVersion: 2.23.16
 name: zipkin
 description: A Zipkin helm chart for kubernetes
 type: application
-version: 0.3.0
+version: 0.4.0
```

Comment:
I think you don't need to change this. The automation will, see https://github.com/openzipkin/zipkin/commit/dffd23618f181a729c3b856f0c49548a2c1b4a44